[2024-04-13 16:05:04] [warning] separator.go:293 p3Chain/core/separator.(*Separator).DoBind Separator Construct: (1) nodes unable to establish P2P communication due to exceptions
[2024-04-13 16:05:09] [warning] separator.go:293 p3Chain/core/separator.(*Separator).DoBind Separator Construct: (1) nodes unable to establish P2P communication due to exceptions
[2024-04-13 16:05:14] [warning] separator.go:293 p3Chain/core/separator.(*Separator).DoBind Separator Construct: (1) nodes unable to establish P2P communication due to exceptions
[2024-04-13 16:05:16] [warning] server.go:631 p3Chain/p2p/server.(*Server).setupConn inbound conn 3023af312ae4d24d [::1]:53124 failed checkpoint posthandshake: Connected to self
[2024-04-13 16:05:16] [warning] server.go:631 p3Chain/p2p/server.(*Server).setupConn static dial conn 3023af312ae4d24d [::1]:20134 failed checkpoint posthandshake: Connected to self
[2024-04-13 16:05:19] [warning] separator.go:293 p3Chain/core/separator.(*Separator).DoBind Separator Construct: (1) nodes unable to establish P2P communication due to exceptions
[2024-04-13 16:05:24] [warning] separator.go:293 p3Chain/core/separator.(*Separator).DoBind Separator Construct: (1) nodes unable to establish P2P communication due to exceptions
[2024-04-13 16:05:29] [warning] separator.go:293 p3Chain/core/separator.(*Separator).DoBind Separator Construct: (1) nodes unable to establish P2P communication due to exceptions
[2024-04-13 16:05:34] [warning] separator.go:293 p3Chain/core/separator.(*Separator).DoBind Separator Construct: (1) nodes unable to establish P2P communication due to exceptions
[2024-04-13 16:05:39] [warning] separator.go:293 p3Chain/core/separator.(*Separator).DoBind Separator Construct: (1) nodes unable to establish P2P communication due to exceptions
[2024-04-13 16:05:44] [warning] separator.go:293 p3Chain/core/separator.(*Separator).DoBind Separator Construct: (1) nodes unable to establish P2P communication due to exceptions
[2024-04-13 16:05:49] [warning] separator.go:293 p3Chain/core/separator.(*Separator).DoBind Separator Construct: (1) nodes unable to establish P2P communication due to exceptions
[2024-04-13 16:05:49] [warning] server.go:631 p3Chain/p2p/server.(*Server).setupConn dyn dial conn 3023af312ae4d24d [::1]:20134 failed checkpoint posthandshake: Connected to self
[2024-04-13 16:05:49] [warning] server.go:631 p3Chain/p2p/server.(*Server).setupConn inbound conn 3023af312ae4d24d [::1]:48018 failed checkpoint posthandshake: Connected to self
[2024-04-13 16:05:54] [warning] separator.go:293 p3Chain/core/separator.(*Separator).DoBind Separator Construct: (1) nodes unable to establish P2P communication due to exceptions
[2024-04-13 16:05:59] [warning] separator.go:293 p3Chain/core/separator.(*Separator).DoBind Separator Construct: (1) nodes unable to establish P2P communication due to exceptions
